Bulkmap fixes

dominikl opened the pull request:
Fixes some issues with the bulkmap config which I messed up following the "tips" from `yamllint`.
Pretty sure `yamllint` will complain again, which I'll just ignore this time, because the bulkmap config works and the format matches the ones from the other projects.
Also removes another unnecessary column from the `annotation.csv`.

dominikl committed:
Removed unnecessary image type column

dominikl committed:
Fixed bulkmap config

dominikl committed:
Fix intendation

sbesson commented:
I haven't applied or tested the modifications here (has it been applied on the testing server?) but the bulkmap config changes match what similar changes I performed in https://github.com/IDR/idr0041-cai-mitoticatlas/commit/1aebc8178d418b19e1f2872c7462dc91cd01e2cf#diff-f0c226fe80f088930dd5c76c90c9c6b5.

dominikl commented:
Oh, already imported the annotations on prod47 (should have used testing first...).

joshmoore commented:
Does that mean we need to look into the re-annotation workflow?

dominikl committed:
Add original assay files (converted to csv format)

dominikl committed:
Update study.txt

dominikl committed:
Regenerated everything due to an error in original csv files

dominikl commented:
Had to regenerate all annotations, filepaths, etc. due to a mistake in the original csv files.

joshmoore commented:
How did you delete the previous annotations?

dominikl committed:
Create top-level bulk.yml containing the import options

sbesson commented on bulk.yml:
```diff
@@ -0,0 +1,13 @@
+output: "yaml"
+# Default columns for the regular screens.
+# This may need to be modified in other bulk files.
+columns:
```
The `columns` in `idr0040-experimentA-bulk.yml` could be fully copied here and removed from the other file for further simplification. Overall :+1:

dominikl commented:
@joshmoore I meant the annotation.csv and filepaths.tsv. I didn't delete anything yet on the server. But I have to delete the whole idr0040 project (on testing and next) and reimport it again. How should I do that, to make sure that the whole project incl. annotations is wiped?

dominikl committed:
Move columns config to top level bulk.yml

dominikl committed:
Just removed the 'TBC' entries from the study.txt

dominikl committed:
Generated annotation.csv/filePaths.tsv from corrected Excel files

dominikl committed:
Fixed whitespace errors in study.txt

dominikl committed:
Fixed Image Name column

dominikl commented:
I think this could be merged now, imported on testing, finally no errors/warnings and looks alright. /cc @sbesson

sbesson reviewed (approved):
From the testing server workflow, the data import and annotation worked without issues. Just mentioned to @dominikl, the assays file should probably be consolidated into a single `idr0040-experimentA-assays.txt` to match existing conventions. The study additions also make sense.

Given the success of the import workflow, I would be inclined to merge this as soon as the assays comment above is addressed.

One of the comments while reviewing the testing resource was the hetereogeneity of the rendering settings across datasets. This should probably be discussed/handled separately.

dominikl committed:
Replace separate assay files with the merged one

dominikl committed:
Added two more little Java "scripts"

dominikl commented:
Removed the separate assay files and replaced with the merged one. Not sure what to do about the rendering settings. For some datasets there is no micromanager rendering settings file. The rendering settings are quite different from dataset to dataset, no sure if we could simply copy and paste one from another dataset.

sbesson commented:
Merging. The outstanding rendering issue can be dealt separately.

sbesson commented:
Oops. Realized too late that the `Experiment Assays File` in the study file needs to be updated to match 4e5af6a. That's a tiny follow-up PR.
